rootfs-builder/fedora/Dockerfile

```diff
@@ -0,0 +1,3 @@
+From fedora:27
```

Comment:
This is identical to the `clearlinux/Dockerfile`. Is there any value in having separate files? I'm wondering if we could have a single file in the `rootfs-builder/` directory and then you could change `rootfs.sh` to do:

```
docker build -f <path_to_the_single_Dockerfile> ...
```

Reply:
I consider we sould have separate files.  Each Dockerfile will install the need dependencies to the script work so for Fedora have a docker image with fedora and golang, for ubuntu an image base in ubuntu with apt.

But agree the image for Clear Linux should be based in Clear Linux in that case.